scripts/install_mujoco.py

```diff
@@ -48,7 +48,7 @@ def install_mujoco(platform, version, mujoco_path):
 
     mujoco_zip_url = f"https://www.roboti.us/download/{mujoco_zip_name}"
 
-    if subprocess.call(["command", "-v", "wget"]) == 0:
+    if subprocess.call(["which", "wget"]) == 0:
```

Comment:
Another way to fix this is to use `shell=True` here:
```suggestion
    if subprocess.call(["command", "-v", "wget"], shell=True) == 0:
```

I'm not an expert on the shell stuff so not sure which one to prefer. Do you have any thoughts?

Reply:
Thanks for looking at this so quickly!
I don't have any preference for one or the other solution either.